### DISCUSSION TITLE
Pre-Release option for Auto-Update

### DISCUSSION THREAD

**Discussion by RobertCajun** [seerge/g-helper#585] [Ideas]
Have an option to let us update within the app to a pre-release version if a newer version is available

**Comment by seerge** [comment]
Pre-releases are marked like that for a reason. It literally means its not ready for everyone to update yet

**Reply by RobertCajun** [reply]
I get that there are some people who will complain because something isnt working in a pre-release, but if there is an option to enable pre-release updates, some may enjoy that.
